integration/data/common.py

```diff
@@ -279,3 +298,9 @@ def verify_async(dev, times, length, count):
     if thread_failed:
         thread_failed = False
         raise Exception("data_verifier thread failed")
+
+
+def for_each_backup_target():
```

Comment:
I just realize you can use pytest fixture for this.

Need to name it `backup_targets` then.